src/main/java/by/bookify/image/controller/ImageController.java

```diff
@@ -14,7 +16,7 @@
 @RequiredArgsConstructor
 public class ImageController {
 
-    private final ImageService imageService;
+    private final GridFsMongoImageService imageService;
 
```

Comment:
And why there are so many realizations? May be you should choose one that suits best

Reply:
if you want only one realization, remove it by yourself